README2.md

```diff
@@ -5,4 +5,7 @@ TEST3
 Testing to push from the local repo to GitHub.
 
 ## TEST 5
-Testing to push from the local repo to Github.
\ No newline at end of file
+Testing to push from the local repo to Github.
+
+### TEST 6
```

Comment:
Successful test of the newly created Feature Branch under the main Branch as part of TEST#6.